config/locales/en.yml

```diff
@@ -179,6 +179,7 @@ en:
         invalid: 'Only letters, numbers, parentheses, hyphen, apostrophe, and # are accepted.'
       payer_tin:
         invalid: EIN must be a 9-digit number. Do not include a dash.
+      payer_tin_ny_invalid: "EIN must be one of: %{valid_tins}"
       phone_number:
```

Comment:
The story said for the error to be the following: The number entered is not an accepted TIN by New York State. Was this a change made later?

Reply:
I've changed this